content/en/service_management/app_builder/components.md

```diff
@@ -0,0 +1,767 @@
+**Value**: string or expression
+
+### Appearance
+
+Intent
+: Controls the color of the button, with colors representing the purpose of the button.<br>
+**Provided values**: default, danger, success, warning
+
+Is Primary
+: Designed to call user attention to the most important action(s) for a given page or workflow.<br>
+**Provided values**: on, off
+
+Is Borderless
+: Removes the border from any button. On hover, it gets a background fill.<br>
+**Provided values**: on, off
+
+Is Loading
+: Shows a loading indicator.<br>
+**Provided values**: on, off
+
+Is Disabled
+: Applies disabled styling and removes interactions.<br>
+**Provided values**: on, off
+
+Is Visible
+: Determines whether the component is visible to the end-user. In edit mode, all components remain visible.<br>
+**Provided values**: on, off
+
+### Events
+
+Event
+: **Value**: click
+
+Reaction
+: **Values**: custom, set component state, trigger query, open modal, close modal, open url, download file
+
+For more information on events, see [Events][1].
+
+### Inspect data
+
+Displays property and value pairs in JSON format.
+
+### Example
+
+To view this component in context, see the [Metrics Explorer & Monitors Builder][2] app blueprint.
+{{% /collapse-content %}} 
+
+
+{{% collapse-content title="Callout value" level="h2" %}}
+Callout value components have the following properties.
+
+### General
+
+Label
+: The text that displays at the top of the component.<br>
+**Value**: string or expression
+
+Value
+: The value that the callout highlights.<br>
+**Value**: string or expression
+
+Unit
+: The unit associated with the value.<br>
+**Value**: string or expression
+
+### Style
+
+Style
+: The visual style of the component.<br>
+**Provided values**: default, success, warning, danger, blue, purple, pink, orange, yellow, red, green, gray, vivid blue, vivid purple, vivid pink, vivid orange, vivid yellow, vivid red, vivid green
+
+Size
+: Responsively sizes the metric so that it is proportional to the sizing of the value.<br>
+**Provided values**: sm, md, lg, xl
+
+### Appearance
+
+Is Loading
+: Shows a loading indicator.<br>
+**Provided values**: on, off
+
+Is Disabled
+: Applies disabled styling and removes interactions.<br>
+**Provided values**: on, off
+
+Is Visible
+: Determines whether the component is visible to the end-user. In edit mode, all components remain visible.<br>
+**Provided values**: on, off
+
+### Inspect data
+
+Displays property and value pairs in JSON format.
+
+### Example
+
+To view this component in context, see the [EC2 Instance Manager][3] app blueprint.
+{{% /collapse-content %}}
+
+
+
+{{% collapse-content title="Checkbox" level="h2" %}}
+Checkbox components have the following properties.
+
+### General
+
+Label
+: The text that displays at the top of the component.<br>
+**Value**: string or expression
+
+Options
+: The list of checkboxes that a user can select from. Minimum number of options is 1.<br>
+**Value**: expression<br>
```

Comment:
For the values that take expressions, it's great that we're giving an example, but I think we should also describe the expected result, because in this case the expression has to conform to the example below - an array of objects, each object consisting of a "label" and "value" key-value pair.

Reply:
PTAL, and this text is reproduced in the other two locations as well, as well as having the others only say they accept expressions (like this one already said)